### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by kolya2731**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
+        return income
+
+    def calculate_washing_price(self, car: tuple) -> float:
+        return round(
+            car.comfort_class * (self.clean_power - car.clean_mark)
+            * self.average_rating / self.distance_from_city_center,
+            1
+        )
+
+    def wash_single_car(self, car: tuple) -> None:
```
Comment (Dimosphen1) [inline app/main.py]: Change your type annotation, `car` should not be a `tuple`. Edit it in all relevant places.

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
```
Comment (Dimosphen1) [inline app/main.py]: Indicate a list of what objects you expect to obtain for `cars` in the type annotation.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested.

**Review by pavlejviki — APPROVED** [review]
Good job!

**Pull request closed without merging**